tests/utils.py

```diff
@@ -180,3 +178,13 @@ def set_dict_element(data, path, value, delimiter='.'):
     path, _, key = path.rpartition(delimiter)
     (get_dict_element(data, path) if path else data)[key] = value
     return data
+
+
+def get_grain(host, key):
+    with host.sudo():
+        output = host.check_output(
+            'salt-call --local --out=json grains.get "{}"'.format(key)
+        )
+        grain = json.loads(output)['local']
```

Comment:
Can be done outside of the `host.sudo()` context

Reply:
Does not seem to work, at least on vagrant box.

Reply:
Oh, strange. OK let's keep it like this and investigate later.